src/commands/friends/FriendCommand.ts

```diff
@@ -105,15 +114,56 @@ export default class FriendCommand extends Command {
     )
 
     if (!friendRequest) {
-      return msg.reply(`**${receiver.name}** has already sent you a friend request.`)
+      return msg.reply(
+        `**${
+          receiver.name
+        }** has already sent you a friend request, or you have already sent them one.`
+      )
     }
 
     return msg.reply(`Sent a friend request to **${receiver.name}**.`)
   }
 
-  async denyFriendRequest (msg: CommandMessage, user: User | string): Promise<Message | Message[]> {
-    // TODO: Delete friend request using API.
-    return msg.reply('This command is not ready yet.')
+  async denyFriendRequest(
+    msg: CommandMessage,
+    user: User | string
+  ): Promise<Message | Message[]> {
+    const rejectee = await getApiUser(user instanceof User ? user.id : user)
+    const sender = await getApiUser(msg.author.id)
+
+    if (!rejectee || !sender) {
+      return msg.reply(
+        'This command requires you to specify a user. Please try again.'
+      )
+    }
+
+    if (rejectee.id === sender.id) {
+      return msg.reply('You can\'t delete a friend request from yourself. That would be silly.')
+    }
+
+    const {
+      data: friendRequests
+    }: { data: UserFriendRequest[] } = await axios.get(
+      `${Plugin.config.api.address}/users/${
+        rejectee.id
+      }/friends/requests?token=${Plugin.config.api.token}`
+    )
+
+    for (const friendRequest of friendRequests) {
+      if (friendRequest.user.id === rejectee.id) {
+        axios.delete(
+          `${Plugin.config.api.address}/users/${rejectee.id}/friends/requests/${
+            friendRequest.id
+          }?token=${Plugin.config.api.token}`
```

Comment:
The url should be `${Plugin.config.api.address}/users/${msg.author.id}/friends/requests/${friendRequest.id}?token=${Plugin.config.api.token}`